coderd/testdata/parameters/modules/main.tf

```diff
@@ -0,0 +1,3 @@
+terraform {}
+
+module "jetbrains_gateway" {}
```

Comment:
https://github.com/aquasecurity/trivy/blob/main/pkg/iac/scanners/terraform/parser/load_module.go#L77-L79

`source` argument is required to load the module